src/OrbitGl/MockTextRenderer.cpp

```diff
@@ -9,12 +9,14 @@
 
 #include <limits>
 
+#include "OrbitGl/BatchRenderGroup.h"
 #include "OrbitGl/CoreMath.h"
+#include "OrbitGl/TextRenderer.h"
 
 namespace orbit_gl {
 
 // Clearing counters also in creation to not duplicate code.
-MockTextRenderer::MockTextRenderer() { Clear(); }
+MockTextRenderer::MockTextRenderer() : TextRenderer() { Clear(); }
```

Comment:
The base class constructor is called anyway in this is not really needed. I assume it's just remnants from a different approach where the text renderer had a constructor argument?

Reply:
Indeed, thanks for spotting. But I also constantly forget which languages automatically call their parent constructor and which don't, to be honest :D